src/App.js

```diff
@@ -1,8 +1,61 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function App() {
+  const [candidates, setCandidates] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  async function getVotes() {
+    try {
+      if(!loading) setLoading(true);
+      const response = await axios.get('http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/candidates');
+      setCandidates(response.data);
+    } catch (e) {
+      setError(e);
+    } finally {
+      if(loading) setLoading(false);
+    }
+  }
+
+  async function doVotes(candidateId) {
+    try {
+      setLoading(true);
+      const response = await axios.get(`http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote?id=${candidateId}`);
+      alert(response.data);
+      getVotes();
```

Comment:
여기에 await 필요 없나?? ....
```suggestion
      await getVotes();
```